vkoves/electrify-chicago
# add docker-entrypoint

# Description

* Adds a `docker-entrypoint.sh` script for running docker commands without shelling into the server. With this, we can run commands like `docker-compose run --rm electrify-chicago bash run_all.sh`
* Updates the Readme with instructions to run commands

Note that you will likely need to run `docker-compose build` before this will work for you.

# Testing Instructions

* run `docker-compose build` to rebuild container
* run `docker-compose run --rm electrify-chicago bash run_all.sh` and confirm it executes as expected

# Checklist:
